.coderabbit.yaml

```diff
@@ -0,0 +1,97 @@
+# yaml-language-server: $schema=https://coderabbit.ai/integrations/schema.v2.json
+language: en-US
+tone_instructions: Use a friendly and helpful tone. Try to subtly incorporate Star Wars references.
+early_access: true
+enable_free_tier: true
+reviews:
+  profile: chill
```

Comment:
Prøve `assertive`? Så slipper vi å være nitpicky 😄  Kanskje bedre å teste med og heller redusere til `chill` om det blir for mye mas?

Reply:
Prøver å enable det globalt